dts/bindings/sensor/memsic,mc3419.yaml

```diff
@@ -21,3 +21,47 @@ properties:
       has two interrupt pins.By default the interrupt are routed
       to interrupt pin 1, by enabled this property interrupt are
       routed to interrupt pin 2.
+
+  lpf-fc-sel:
+    type: int
+    default: 0
+    enum:
+      - 0
+      - 9
+      - 10
+      - 11
+      - 13
+    description: |
+      Select LPF cut-off frequency.
+      Possible values and corresponding values are listed below
+      0 => disable LPF
+      9 => LPF with fc set to IDR/4.255
+      10 => LPF with fc set to IDR/6
+      11 => LPF with fc set to IDR/12
+      13 => LPF with fc set to IDR/16
+      Default is LPF disabled (reset value of the register).
```

Comment:
This is IMHO is a pretty bad DT usage. If I read in DT `lpf-fc-sel = <11>` I have no clue at all what's going on. Reading bindings isn't clear enough either, eg, what is IDR?

I'd expect something like:

- no property specified -> disable LPF
- `lpf-fc-scale` -> scale value

Reply:
DT explanation is reworded, let me know if it more clear now. IDR was explained in decimation-rate property, i have included it in this too.
If no property is specified then i have to use DT_INST_PROP_OR with 0, isn't it more descriptive if we use 0 as default value in DT definition itself then silently using it?
And `lpf-fc-sel` are already explained as it is in the datasheet. Do you want me to add datasheet reference?

Reply:
@gmarull let me know your thoughts on this too.

Reply:
I guess then we need some sort of human readable defines on a header.